Source/References/ItemsHoldr-0.2.1.ts

```diff
@@ -1117,6 +1122,13 @@ module ItemsHoldr {
         }
 
         /**
+         * Toggles whether autoSave is true or false.
+         */
+        toggleAutoSave(): void {
+            this.autoSave = this.autoSave ? false : true;
```

Comment:
Yes. A more concise way of doing that would be `this.autoSave = !this.autoSave;`.